ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/mlag_ibgp_peering_vrfs.schema.yml

```diff
@@ -0,0 +1,26 @@
+# yaml-language-server: $schema=../../../../plugins/plugin_utils/schema/avd_meta_schema.json
+# Line above is used by RedHat's YAML Schema vscode extension
+# Use Ctrl + Space to get suggestions for every field. Autocomplete will pop up after typing 2 letters.
+type: dict
+keys:
+  mlag_ibgp_peering_vrfs:
+    documentation_options:
+      filename: Fabric Variables
+      table: MLAG Settings
+    type: dict
+    description: |
+      On mlag leafs, an SVI interface is defined per vrf, to establish iBGP peering (required when mlag leafs in topology)
+      The SVI id will be derived from the base vlan defined: mlag_ibgp_peering_vrfs.base_vlan + (vrf_id or vrf_vni) - 1
+      Depending on the values of vrf_id / vrf_id it may be required to adjust the base_vlan to avoid overlaps or invalid vlan ids.
```

Comment:
```suggestion
      Depending on the values of vrf_id / vrf_vni it may be required to adjust the base_vlan to avoid overlaps or invalid vlan ids.
```